clippy_config/src/msrvs.rs

```diff
@@ -17,7 +17,7 @@ macro_rules! msrv_aliases {
 
 // names may refer to stabilized feature flags or library items
 msrv_aliases! {
-    1,76,0 { PTR_FROM_REF }
+    1,76,0 { PTR_FROM_REF, C_STR_LITERALS }
```

Comment:
It looks like they're planning to stabilize it in 1.77: 

> We discussed this in the triage call today and agreed, given that https://github.com/rust-lang/rust/pull/119172 has landed, that we were happy to see this restabilize in Rust 1.77
>
> https://github.com/rust-lang/rust/issues/105723#issuecomment-1919576337

On nightly these strings already seem to work, so I think it's safe to merge this, once the version is updated :)

Reply:
oh right, oops, of course 😅